code/modules/projectiles/updated_projectiles/ammunition.dm

```diff
@@ -19,6 +19,7 @@ They're all essentially identical when it comes to getting the job done.
 	throw_speed = 2
 	throw_range = 6
 	var/default_ammo = /datum/ammo/bullet
+	var/overcharge_ammo = /datum/ammo/bullet //Generally used for energy weapons
```

Comment:
This is a hack and it should be using the energy system.

Reply:
What energy system? You mean the one from TG that I'm definitely not porting over?

Reply:
If you're not willing to port or rewrite large systems yourself don't even bother adding these kinds of features which if done improperly introduce large amounts of code debt.